protopipe_grid_interface/scripts/submit_jobs.py

```diff
@@ -210,6 +220,9 @@ def main():
     # GRID
     outdir = os.path.join(cfg["GRID"]["outdir"], config_name)
     n_file_per_job = cfg["GRID"]["n_file_per_job"]
+    if "n_file_per_job" in switches:
+        n_file_per_job = int(switches["n_file_per_job"])
+
```

Comment:
else? we should have a default argument (I'd say 10 files per job is always good choice)

Reply:
yeah we should have defaults everywhere (or raise errors if an option needs to be specified always for safety)

Reply:
1 is fine